frontend-student/src/components/myCourses/MyCourses.jsx

```
@@ -0,0 +1,258 @@
+                  </svg>
+                  {course.rating}
+                </div>
+              )}
+            </div>
+            <div className='p-6'>
+              <h2 className='text-xl font-bold text-gray-900 mb-2'>
+                {course.title}
+              </h2>
+              <div className='flex justify-center items-center w-full'>
+                <Link to={`/enrolled/${course.id}`}>
+                  <button className='mt-4 px-5 py-2 bg-blue-600 text-white rounded-lg hover:bg-blue-700 transition-colors duration-300 flex items-center'>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)